packages/image_picker/android/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,17 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="io.flutter.plugins.imagepicker">
-</manifest>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
```

Comment:
Do people still need to add this to their own app as well es described in the README? https://github.com/flutter/plugins/tree/f2210c5403057b9ec8038b00c4b867f55c9b038a/packages/image_picker#android

Or should that be updated as well?

Reply:
These permissions should get merged to the main app's Android Manifest according to [the Manifest Merger documentation](https://developer.android.com/studio/build/manifest-merge.html).

I updated the Android-specific instructions in the README.